### DISCUSSION TITLE
Old html view

### DISCUSSION THREAD

**Discussion by juliocpmelo** [jplag/JPlag#397] [Q&A] [answered]
Is it possible to have the old html view in the current version?

I'm a jplag hard user since 2019, the current viewer using vue is not finished (at least I was not able to see an useful comparison result last week when i tested it) and I'm having some problems to build the [v2.12.1-SNAPSHOT](https://github.com/jplag/JPlag/releases/tag/v2.12.1-SNAPSHOT).  (I did some small changes to make it easier to spot and fix the parser errors).

**Comment by tsaglam** [comment, marked as answer]
Yes, v3.0.0 still uses the legacy report in a slightly modified. So either download the [v3.0.0 release](https://github.com/jplag/JPlag/releases/tag/v3.0.0) or the [legacy release](https://github.com/jplag/JPlag/releases/tag/v2.12.1-SNAPSHOT).
If you want to build your own version, check out the [tagged commit for the v3.0.0 version](https://github.com/jplag/JPlag/tree/v3.0.0) and go from there.
Just out of interest, if you need to build JPlag yourself, what is your current use case?

For other readers of this thread: A beta version of the new report viewer can be found here: https://jplag.github.io/JPlag/

**Reply by juliocpmelo** [reply]
Ok thanks I'll give it another try later today. When I generated the results I only got some json files. That I used with the new viewer and was able to see some overall info and even the compared files, but not the similarity between the files, which was the important part for me...

**Comment by juliocpmelo** [comment]
Just to confirm, it worked fine thanks @tsaglam.
